core/app/flags/experimental.go

```diff
@@ -26,4 +26,5 @@ var (
 	EnableFrameLifecycle = flag.Bool("experimental-enable-frame-lifecycle", false, "Enable the experimental feature Frame Lifecycle.")
 	EnableVulkanTracing  = flag.Bool("experimental-enable-vulkan-tracing", false, "Enable the experimental feature Vulkan tracing.")
 	EnableAngleTracing   = flag.Bool("experimental-enable-angle-tracing", false, "Enable the experimental feature ANGLE tracing.")
+	EnablePerfTab        = flag.Bool("experimental-enable-perf-tab", false, "Enable the experimental feature Counter performance tab.")
```

Comment:
Is this flag used anywhere in the go side? I don't think we need it on the server, "Tab" is very much UI specific :)

Reply:
It's not yet used in the server side. But I think when I move the counter sample digestion logic to gapis, this flag might be put into use then. 
Yes, maybe this flag should be given a more neutral name, maybe "experimental-enable-counter-perf"? Or any good naming idea? The only worry I have is that I may introduce too many alias around this project. Will do the renaming if necessary in the following logic movement PR.